### DISCUSSION TITLE
Add support for im.mode = "LA" in PdfImagePlugin.py

### DISCUSSION THREAD

**Discussion by mnmtz** [python-pillow/Pillow#7293] [Ideas]
Please add support for images with mode "LA"  in the PdfImagePlugin:

```python
elif im.mode == "LA":
    filter = "JPXDecode"
    colorspace = PdfParser.PdfName("DeviceGray")
    procset = "ImageB"  # grayscale
```

**Comment by radarhere** [comment]
Could you provide some background for why you chose JPXDecode, DeviceGray and ImageB?

**Reply by mnmtz** [reply]
I want to replace a png image with mode "LA" with a compressed jp2k image in a PDF file while keeping the alpha channel to reduce the size of the PDF. The provided code was a temporary fix to get this done.

**Reply by radarhere** [reply]
Ok, see what you think of #7299

**Reply by radarhere** [reply]
#7299 has now been merged
